pytorch_lightning/__init__.py

```diff
@@ -10,9 +10,10 @@
 __docs__ = "PyTorch Lightning is the lightweight PyTorch wrapper for ML researchers." \
            " Scale your models. Write less boilerplate."
 
-from logging import getLogger
+import logging as python_logging
```

Comment:
Is there a special reason you cannot import just the two needed functions?

Reply:
it feels cleaner to alias, which still avoids the namespace collision

Reply:
```
>>> import pytorch_lightning
>>> pytorch_lightning.logging.__path__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'pytorch_lightning' has no attribute 'logging'
>>> pytorch_lightning.loggers.__path__
['/Users/jeremyjordan/Documents/Code/pytorch-lightning/pytorch_lightning/loggers']
>>> pytorch_lightning.python_logging.__path__
['/Users/jeremyjordan/anaconda/envs/lightning/lib/python3.7/logging']
```

Reply:
Sure, but why would you need to import whole package while you need just one function...